.github/workflows/acceptance_tests.yaml

```diff
@@ -96,8 +118,8 @@ jobs:
           ${CI_DIR}/autoscaler/scripts/run-acceptance-tests.sh
 
   acceptance_test_broker:
-    needs: acceptance_test_app
-    name: Broker acceptance tests
+    needs: [ deploy_autoscaler ]
```

Comment:
Im still running these in parrallel.... lets see if this causes problems. If it does we can make it serialised.